python/ext/third_party/zi_lib/gpl-3.0.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Zlib is linked to our ingest process. I don't think we're distributing that in a way that would violate GPL, so it should be ok. We may want to replace this with the Google meshing code anyway to avoid duplication.